open-policy-agent/service-account-mgmt/serviceaccount_automounttoken_prevention/README.md

```diff
@@ -0,0 +1,234 @@
+# Policy Explainer & Testing - ServiceAccount Automount Token Prevention
+
+The policy disallows creating serviceAccounts that does not include the following key:value pair in its manifest: `"automountServiceAccountToken": false`
+
+It is useful (security-wise) so that by default services' pods won't be able to mount the token of the serviceAccount which deployed them if they aren't meant to. In case that a serviceAccount token is required it can be excluded.
+
+The required procedure to deploy the policy:
+
+1. Add the subscription for the gatekeeper operator
+2. Generate the relevant gatekeeper object CR
+3. Deploy the template & constraint yamls that define the policy
+* Note that any openshift-* & kubernetes-* default namespaces are excluded
+4. Deploy the test serviceAccount objects to make sure the policy works as expected
+
+`You should be logged in as cluster-admin privilaged user`
+
+## Add the subscription for the gatekeeper operator
+
+```bash
+cat > subscription.yaml << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gatekeeper-operator-product
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: gatekeeper-operator-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+oc create -f subscription.yaml
+oc get subscriptions.operators.coreos.com -A | grep -i gatekeeper
+```
+
+
+## Generate the relevant gatekeeper object CR
```

Comment:
@tommeramber @michaelkotelnikov 
I think it's a general guideline on OPA Gatekeeper which is irrelevant to specific policy readme and makes the README too long. 

In my opinion it is better to focus on policy implementation, and example without general gatekeper configuration how-to.

Reply:
Fixed